docs/rule/a11y/alt-text.md

```diff
@@ -0,0 +1,79 @@
+## a11y-alt-text
+
+Enforce that all elements that require alternative text have meaningful information to relay back to the end user. This is a critical component of accessibility for screenreader users in order for them to understand the content's purpose on the page. By default, this rule checks for alternative text on the following elements: `<img>`, `<area>`, `<input type="image">`, and `<object>`.
+
+
+This rule **forbids** the following:
+
+
+### `<img>`
+
+An `<img>` must have the `alt` prop set with meaningful text or as an empty string to indicate that it is an image for decoration.
```

Comment:
the rule is that if it's a meaningful image, it should have a non-empty alt value. If it's not a meaningful image, it should have an empty alt attribute value and have the role of presentation or none. 

Okay to have:
`<img alt="" role="none" src="zoey.jpg" />`
OR
`<img alt="" role="presentation" src="zoey.jpg" />`
OR
`<img alt="a stylized graphic of a female hamster" src="zoey.jpg" />`

Not okay: 
`<img src="zoey.jpg"/>`
OR 
`<img alt="" src="zoey.jpg" />`